src/components/backend-ai-credential-view.ts

```diff
@@ -78,6 +78,8 @@ export default class BackendAICredentialView extends BackendAIPage {
   @property({type: Number}) selectAreaHeight;
   @property({type: Boolean}) enableSessionLifetime = false;
 
+  static readonly MAX_INT32 = 0x7FFFFFFF;
```

Comment:
This same constant is defined in multiple places. How about taking this out to a common module?